src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -3310,70 +3335,347 @@ new_readdir(DIR *dirp)
 	return &mydir->ents[mydir->num_ents];
 }
 
+extern char **__environ;
+
+/* This is the number of environmental variables that would be forced to set in child process. */
+#define N_ENV_APPEND (7)
```

Comment:
Should this be forwarding n D_LOG_FILE as well?

Reply:
Thank you! I will add "D_LOG_FILE". Shall we add "DD_MASK", "DD_SUBSYS", and "D_LOG_MASK" too?

Reply:
Yes, I think so.

It would be a good idea to re-write this code to use and array (possilby you had it like that before?) so that adding new ones is easy as I'm sure we're going to find new entries to add to the list at some point.  I think LD_PRELOAD is the only one that needs a special case here, the rest all copy from the current process I assume?

Reply:
Thank you! I will look into rewriting the code here as you suggested.

Reply:
I just updated the code to loop over a given env list to append. Thank you!